2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,8 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let currency = {};
+  return (currency = Object.fromEntries(countryCurrencyCodes));
```

Comment:
This is a good use of Object.fromEntries! Maybe you've already seen this, but you can read more about this here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries
One small question - do you need the currency variable here? Are you able to return the output. of the fromEntries function directly?